### ISSUE TITLE
Bug StringIO

### ISSUE THREAD

**Issue by wannaphong** [theopolisme/webcam-streamer#1]
I using CPython 3.4 64 bit on Windows 8.1 64 bit.
webcam-streamer 1.0.5
I using

> webcam-streamer
> but.
> 
> ``` python
> F:\>webcam-streamer
> Traceback (most recent call last):
>   File "C:\py34\lib\runpy.py", line 170, in _run_module_as_main
>     "__main__", mod_spec)
>   File "C:\py34\lib\runpy.py", line 85, in _run_code
>     exec(code, run_globals)
>   File "C:\py34\Scripts\webcam-streamer.exe\__main__.py", line 5, in <module>
>   File "C:\py34\lib\site-packages\webcamstreamer\launcher.py", line 7, in <modul
> e>
>     import StringIO
> ImportError: No module named 'StringIO'
> ```
> 
> https://stackoverflow.com/questions/11914472/stringio-in-python3

**Comment by theopolisme** [comment]
Thanks!
(Fixed by @wannaphongcom in #2)

**Comment by escrowdis** [comment]
It's still not solved, try replace by
```
try:
    from StringIO import StringIO
except ImportError:
    from io import StringIO
```
